package.json

```diff
@@ -21,11 +21,13 @@
     "url": "https://github.com/Aircoookie/WLED/issues"
   },
   "homepage": "https://github.com/Aircoookie/WLED#readme",
+  "devDependencies": {
+    "nodemon": "^3.0.1"
```

Comment:
Why only move nodemon to devdependencies? In theory, all of the dependencies are dev only, nothing gets put on the device itself after build.

Reply:
There's a difference in the dependencies used for "development", and the tools used to "build". The dependencies left in the `dependencies` declaration (as opposed to `devDependencies` declaration), as used during the build process.

Reply:
That's one way of seeing it. It could work. However, I am more used to see it as`dependencies` is anything required in production environments while `devDependencies` is anything needed for the project development, which includes build tools.

In any cases, there is no real point in implementing a separation between devDependencies or normal dependencies right now as we do not use any library in production. All of them are either build tools or dev tools and so we do not require installing only a certain group. They will all not be included in prod.

Reply:
This change "grouping" could speed up the build, as 'nodemon' wouldn't be needed for the build - and it's likely the build tool isn't using the `devDependencies`.

This PR is also for a version bump of nodemon.